src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -225,6 +225,10 @@ public final class Constants {
 
     public static final int HAZELCAST_QUIZ_EXERCISE_CACHE_SERIALIZER_ID = 1;
 
+    public static final String HAZELCAST_PLAGIARISM_PREFIX = "plagiarism-";
```

Comment:
I guess we should stay consistent in this PR, but in the future / follow-up I think these are another set of constants that should rather be located in their own classes (PlagiarismCacheService, etc).